examples/single-account/base.tf

```diff
@@ -0,0 +1,30 @@
+# terraform apply --target aws_dynamodb_table.basic-dynamodb-table
+resource "aws_dynamodb_table" "basic-dynamodb-table" {
```

Comment:
If possible don't use resource type in the resource name.

Reply:
renamed